packages/snaps-utils/src/manifest/validation.ts

```diff
@@ -47,11 +47,26 @@ const FORBIDDEN_PATHS: string[][] = FORBIDDEN_COIN_TYPES.map((coinType) => [
   `${coinType}'`,
 ]);
 
-const BIP32_INDEX_REGEX = /^\d+'?$/u;
+const BIP32_INDEX_REGEX = /^(\d+)'?$/u;
+
+const isValidBip32Path = (path: string[]) => {
```

Comment:
Please use `isValidBIP32PathSegment` from `key-tree@7.1.0` (https://github.com/MetaMask/key-tree/blob/main/src/utils.ts#L182).

Currently waiting for approval to be published.

Reply:
Might as well takeover the Pr lol

Reply:
Sure 😅